test/src/Provider/AbstractProviderTest.php

```diff
@@ -507,4 +506,32 @@ public function testInvalidAccessTokenMethod()
         $provider->setAccessTokenMethod('PUT');
         $provider->getAccessToken('authorization_code', ['code' => 'mock_authorization_code']);
     }
+
+    private function getMethod($class, $name)
+    {
+        $class = new \ReflectionClass($class);
+        $method = $class->getMethod($name);
+        $method->setAccessible(true);
+        return $method;
+    }
+
+    public function getAppendQueryProvider()
```

Comment:
I think the rest of the data providers use the form `providerNameOfTest` or similar, with `provider` always used as the prefix.